test/test_sparse.py

```diff
@@ -1280,6 +1280,19 @@ def test_is_nonzero(self):
         self.assertFalse(torch.sparse_coo_tensor(([0, 0],), (-1., 1.), (1,)).is_nonzero())
         self.assertTrue(torch.sparse_coo_tensor(torch.zeros(0, 1), 12.3, []).is_nonzero())  # scalar sparse tensor
 
+    def test_nyi_functions(self):
```

Comment:
It's not that they're not yet implemented... they'll never be implemented, right?